controllers/workspace/status.go

```diff
@@ -320,3 +320,21 @@ func checkForFailingTimeout(workspace *common.DevWorkspaceWithConfig) (isTimedOu
 	}
 	return false, nil
 }
+
+func fmtStartTimeoutMessage(workspace *common.DevWorkspaceWithConfig) error {
+	status := workspaceConditionsFromClusterObject(workspace.Status.Conditions)
+	latestCondition := status.getFirstFalse()
+	if latestCondition != nil {
+		return fmt.Errorf("Devworkspace failed to progress past step '%s' for longer than timeout (%s)",
```

Comment:
The go linter is complaining about these error messages starting with a capital letter (which IMO is fine for this case).

Though, it's probably worth changing the capitalization on `Devworkspace` -> `DevWorkspace` for consistency (E.g. the workspace status info messages write it `DevWorkspace`)